package.json

```diff
@@ -7,22 +7,23 @@
   "license": "MIT",
   "private": false,
   "scripts": {
-    "build:env": "sudo apt-get install libzmq3-dev -y && sudo apt install libsocketcan-dev -y && sudo apt install pkg-config -y && sudo apt install python3-dev -y && sudo apt install python3-pip -y && sudo pip3 install numpy",
-    "csp:clone": "rm -rf libcsp && git clone --branch master https://github.com/Albertasat/libcsp.git && cd libcsp && git clone git@github.com:AlbertaSat/ex2_sdr.git",
-    "csp:configure": "cd libcsp && python3 waf configure --with-os=posix --enable-SDR --enable-can-socketcan --enable-rdp --enable-crc32 --enable-hmac --enable-xtea --with-loglevel=debug --enable-debug-timestamp --enable-python3-bindings --with-driver-usart=linux --enable-if-zmqhub --enable-examples",
+    "install_dependencies": "sudo apt-get install libzmq3-dev -y && sudo apt install libsocketcan-dev -y && sudo apt install pkg-config -y && sudo apt install python3-dev -y && sudo apt install python3-pip -y && sudo pip3 install numpy",
+    "csp:clone": "rm -rf libcsp && git clone https://github.com/Albertasat/libcsp.git && cd libcsp && git clone git@github.com:AlbertaSat/ex2_sdr.git",
+    "csp:configure": "cd libcsp && python3 waf configure --with-os=posix --enable-SDR --enable-rdp --enable-crc32 --enable-hmac --enable-xtea --with-loglevel=debug --enable-debug-timestamp --enable-python3-bindings --with-driver-usart=linux --enable-examples",
```

Comment:
Note that there are still things like --enable-crc32 --enable-xtea and even --enable-rdp that we don't really need (although they don't really hurt either)

Reply:
They're good to leave since it doesn't hurt anyone